new-dti-website/components/ui/button.tsx

```diff
@@ -35,14 +35,20 @@ const buttonVariants = cva(
 export interface ButtonProps
   extends React.ButtonHTMLAttributes<HTMLButtonElement>,
     VariantProps<typeof buttonVariants> {
+  ariaLabel?: string;
   asChild?: boolean;
 }
 
 const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
-  ({ className, variant, size, asChild = false, ...props }, ref) => {
+  ({ className, variant, size, asChild = false, ariaLabel, ...props }, ref) => {
     const Comp = asChild ? Slot : 'button';
     return (
-      <Comp className={cn(buttonVariants({ variant, size, className }))} ref={ref} {...props} />
+      <Comp
```

Comment:
Awesome this looks great! Thanks for doing this! After we get this in I can add the variants to this so we can omit the button styles in the global css! Thanks again Chris!

Also just for future reference I think it'd be best to just use this component now instead of the button tag! I don't think we have to do that for this PR though!